core/kubernetes/main/openhim/openhim-mongo-deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: openhim-mongo-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: openhim-mongo
+  template:
+    metadata:
+      labels:
+        component: openhim-mongo
+    spec:
+      volumes:
+        - name: openhim-mongo-storage
+          persistentVolumeClaim:
+            claimName: openhim-mongo-volume-claim
+      containers:
+        - name: openhim-mongo
+          image: mongo:4.2
+          ports:
+            - containerPort: 27017
+          volumeMounts:
+            - name: openhim-mongo-storage
+              mountPath: /data/db
+              subPath: mongo
```

Comment:
I think this subPath can be removed as we now make use of the configmap volume

Reply:
This mongo deployment doesn't have a configmap - just a persistent volume claim

Reply:
ok cool makes sense :+1: